tests/CoreFX/TopN.CoreFX.Windows.issues.json

```diff
@@ -0,0 +1,3686 @@
+[
+    {
+        "name": "Canonicalization.Tests",
+        "enabled": true,
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": null
+        }
+    },
+    {
+        "name": "Common.Performance.Tests",
+        "enabled": true,
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": null
+        }
+    },
+    {
+        "name": "Common.Tests",
+        "enabled": true,
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": [
+                {
+                    "name": "System.Tests.StringTests.IndexOfAny_NullAnyOf_ThrowsArgumentNullException",
+                    "reason": "System.MissingMethodException System.MissingMethodException : Method not found: 'Void System.AssertExtensions.Throws(System.String, System.String, System.Func`1<System.Object>)'."
```

Comment:
Why are we hitting these exceptions? It suggests that we are picking up wrong or inconsistent corefx bits somewhere.

Reply:
This seems to be XUnite internals.
I think likely due to a version mismatch in xunit dependencies between the buildtools xunit runner and the built assemblies. 
I'll confirm, update the buildtools dependency on XUnit and that should be published to MyGet.

Reply:
I was wrong - this is CoreFX.Private.TestUtilities. Updating it fixed the issue. I will remove the exclusions.

Reply:
I was wrong - this is CoreFX.Private.TestUtilities (the System.* namespace should have been a giveaway). Updating it fixed the issue. I will remove the exclusions.